code/modules/research/designs/bluespace_designs.dm

```diff
@@ -5,7 +5,7 @@
 	name = "Artificial Bluespace Crystal"
 	desc = "A small blue crystal with mystical properties."
 	id = "bluespace_crystal"
-	req_tech = list("bluespace" = 4, "materials" = 6)
+	req_tech = list("bluespace" = 2)
```

Comment:
Oh yeah, this one is totally on purpose. It's actually called out in the original design. I don't like the inconsistency, but I have no problem with changing this one back. I have no problem with changing _any_ of the ones you guys have a problem with back.